pymc/distributions/shape_utils.py

```diff
@@ -59,8 +59,9 @@ def to_tuple(shape):
 
     Returns
     -------
-    If `shape` is None, returns an empty tuple. If it's an int, (shape,) is
-    returned. If it is array-like, tuple(shape) is returned.
+    shape: 
```

Comment:
The info that must be here according to numpydoc is the return type. Giving a name to the returned object is optional. Here is should be one of:

```
tuple
shape : tuple
```

Note that when only the type is used, there are no colons, if a name is also given, the colon needs a space before and after it. Ref: https://numpydoc.readthedocs.io/en/latest/format.html#returns